DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/loop_carried_dependency/sample.json

```diff
@@ -13,6 +13,8 @@
       {
         "id": "fpga_emu",
         "steps": [
+          "dpcpp --version",
+          "cd ../../..",
           "mkdir build",
           "cd build",
           "cmake ..",
```

Comment:
'cmake ..' is no longer the correct path.  You need to run cmake ./Tutorials/DesignPatters/loop_carried_dependency (note this will be different for each test, which will complicate the change you need to make)